doc/classes/DisplayServer.xml

```diff
@@ -900,6 +900,11 @@
 			Fullscreen window mode. Note that this is not [i]exclusive[/i] fullscreen. On Windows and Linux, a borderless window is used to emulate fullscreen. On macOS, a new desktop is used to display the running project.
 			Regardless of the platform, enabling fullscreen will change the window size to match the monitor's size. Therefore, make sure your project supports [url=$DOCS_URL/tutorials/rendering/multiple_resolutions.html]multiple resolutions[/url] when enabling fullscreen mode.
 		</constant>
+		<constant name="WINDOW_MODE_EXCLUSIVE_FULLSCREEN" value="4" enum="WindowMode">
+			Exclusive fullscreen window mode. This mode is implemented on Windows only. On other platforms, it is equivalent to [constant WINDOW_MODE_FULLSCREEN].
+			Only one window can enter exclusive fullscreen mode, when window is in exclusive fullscreen mode, child window can not be displayed.
```

Comment:
```suggestion
			Only one window can enter exclusive fullscreen mode. When a window is in exclusive fullscreen mode, child windows can not be displayed.
```

Reply:
BTW, what happens if a user sets exclusive mode to one window, and then attempts to set it to another? Does the second `window_set_mode` call override the first, or is the second window's mode ignored?

Reply:
It's a bit imprecise description. It is possible to set exclusive mode for multiple windows (and useful to do it on multiple screens). Setting another one to exclusive mode is fine, it will bring it to the front and hide the first one.
I guess I'll change it to:
```
Only one window in exclusive fullscreen mode at a time can be simultaneously visible on a screen.
```

Reply:
Sounds good, maybe:
```
Only one window in exclusive fullscreen mode can be visible on a given screen at a time. If multiple windows are in exclusive fullscreen mode for the same screen, the last one being set to this mode takes precedence.
```